xc/common/cmake/arch_define.cmake

```diff
@@ -161,15 +157,15 @@ function(ADD_XC_ARCH_DEFINE)
         --vpr_capnp_schema_dir ${VPR_CAPNP_SCHEMA_DIR} \
         --route \${OUT_ROUTE} \
         --bitread $<TARGET_FILE:bitread> \
-        --bit_file \${OUT_BIN} \
-        --fasm_file \${OUT_BIN}.fasm \
+        --bit_file \${OUT_BITSTREAM} \
+        --fasm_file \${OUT_BITSTREAM}.fasm \
         --iostandard ${DEFAULT_IOSTANDARD} \
         --drive ${DEFAULT_DRIVE} \
         \${PCF_INPUT_IO_FILE} \
         --eblif \${OUT_EBLIF} \
         --top \${TOP} \
         \${OUT_BIT_VERILOG} \${OUT_BIT_VERILOG}.xdc"
-
+    NO_BIT_TO_BIN
```

Comment:
This doesn't do what you want.  I believe all the targets that install bitstream's on targets will break.

Reply:
Actually, what I did here is to have only one step for the bitstream generation. Previously, as there were two separate steps (fasm2frames and xc7frames2bit, the FASM_TO_BIT and BIT_TO_BIN were required.

Given that with xcfasm we can have only one step, the BIT_TO_BIN is no longer required. All the xc7 `<test>_bin` targets have been deleted and substituted with the `<test>_bit`.

Reply:
I understand what you are saying, but https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/common/cmake/devices.cmake#L1658 will break the way it is now.

Reply:
Ok, now I get it. I guess this is uncaught by CI. One way to fix it would be to modify devices.cmake to set the correct dependency. E.g.: if there is no BIT to BIN the PROG target takes the OUT_BITSTREAM as dependency

Reply:
I wonder if we just need a BIT_IS_BIN, and passthrough the right variables.  It'll remove some of the more random changes here.

Reply:
I am uncertain of what the random changes are. IMO it would be better to have the OUT_BIN set to default to point at the OUT_BITSTREAM, and, if the BIT_TO_BIN step is enabled, than the OUT_BIN will get overridden.